docs/dev/clusteroperator.md

```diff
@@ -97,3 +97,22 @@ If an error blocks reaching 4.0.1, the conditions might be:
 * `Progressing` is true with message `Unable to apply 4.0.1: a required object is missing`
 
 The progressing message is the first message a human will see when debugging an issue, so it should be terse, succinct, and summarize the problem well.  The failing message can be more verbose. Start with simple, easy to understand messages and grow them over time to capture more detail.
+
+
+#### Conditions and Install/Upgrade
+
+Conditions determine when the CVO considers certain actions complete, the following table summarizes what it looks at and when.
+
+
+| operation | version | available | degraded | progressing | 
+|-----------|---------|-----------|----------|-------------|
+| Install completion[1] | current(whatever was being installed) | true | any | any
+| Begin upgrade | any | any | any | any
+| Begin upgrade (w/ force) | any | any | any | any 
+| Upgrade completion[2]| newVersion(target version for the upgrade) | true | false | false
+
+[1] Install cannot proceed to the next cluster operator until the previous cluster operator achieves these conditions 
```

Comment:
during install we allow all operators to be created at once _ same level_
cc @smarterclayton

Reply:
ah yes, i'll clarify.

Reply:
@abhinavdahiya fyi when I talked to @smarterclayton he said we created all operators at once, period (regardless of runlevel), so i updated the doc to reflect that.